torch_geometric/nn/nlp/llm.py

```diff
@@ -117,6 +107,102 @@ def _encode_inputs(
         return (batch_size, questions, context, eos_user_tokens, bos_embeds,
                 pad_embeds)
 
+    def _label_input_ids(self, i, label, eos_tokens):
```

Comment:
Can we also add typing to the arguments in these functions? Thanks.

Reply:
sure, im ooto until tues aug 27, but can circle back then if still needed